lib/src/group_changes.dart

```diff
@@ -321,16 +350,27 @@ String htmlPage(List<SummaryData> data, List<String> hashes,
       }
       page.write("</div>");
 
+      String formattedResult(Result result) {
+        if (result.newFailure) {
+          return "New test <strong>${result.current}</strong>"
+              " (expected ${result.expected})";
```

Comment:
You really should use a package for escaping text for insertion into HTML, and use a proper split between the data model and rendering of HTML.

Reply:
Noted.